brave/brave-core
# Polish new extension menu view UI

Part of https://github.com/brave/brave-browser/issues/37583

<img width="469" alt="image" src="https://github.com/brave/brave-core/assets/5474642/6d48039a-63fe-4c49-805b-3dd58157ddbf">




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Enable flag brave://flags/#extensions-menu-access-control
* See new extensions menu
